packages/app-admin-ui/index.js

```diff
@@ -120,16 +122,14 @@ class AdminUIApp {
 
   getAdminUIMeta(keystone) {
     // This is exposed as the global `KEYSTONE_ADMIN_META` in the client.
-    const { adminPath, apiPath, graphiqlPath, pages, hooks } = this;
+    const { adminPath, apiPath, graphiqlPath } = this;
     const { signinPath, signoutPath } = this.routes;
     const { lists, name } = keystone.getAdminMeta({ schemaName: this._schemaName });
     const authStrategy = this.authStrategy ? this.authStrategy.getAdminMeta() : undefined;
     return {
       adminPath,
       apiPath,
       graphiqlPath,
-      pages,
```

Comment:
this is also part of `AdminMetaProvider` which I use in several PR as well. would be helpful in customising admin-ui.

Reply:
I think that's the old `pages` config. People should be using the new `pages` hook instead of the direct config option, shouldn't they?

Reply:
yeah, the hooks should be enough.

Reply:
Indeed. Might also be worth adding a deprecation/removal notice for `pages` since technically its functionality was never removed until now.